content/en-us/tutorials/fundamentals/coding-5/intro-to-dictionaries.md

```diff
@@ -10,10 +10,10 @@ Dictionaries are tables that associate names or "\*keys\*\*" with a value instea
 Example:
 
 ```lua title="Lua dictionary syntax"
- local pet = {
-    Name = "Bobbie",
-    Type = "Dog",
- }
+local pet = {
+   Name = "Bobbie",
+   Type = "Dog",
```

Comment:
I think we use tabs (or 4 spaces?), but not 3 spaces, cc @Roblox/documentation for style review

applies here and to below changes in this file